### ISSUE TITLE
508-defect-2 [COGNITION]: Heading levels SHOULD increase by one in a logical order

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#17012]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:**  Josh Kim

## User Story or Problem Statement

As a screen reader user, I want to navigate the hierarchy of the page content using heading levels to save time and frustration.

## Details

Headings on a web page should progress, increasing by one, in sections of content. There are a number of headings throughout the pages that don't adhere to a logical heading order of H1, H2, H3, H4, etc.

For example, it the page was written as an outline, the following hierarchy would not make sense as "Check your payments..." simply describes/rewords "Your VA Payments."
1. Your VA Payments
   1. Check your payment history for your VA disability compensation, pension, and education benefits.
      1. Payments you received

## Acceptance Criteria

- [ ] Headings progress one by one in a logical order.
- [ ] Current `h2` is converted to a `p` under the `h1`.
- [ ] Frequently Asked Questions `h2` is added to the page.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging and production

## Steps to Recreate

1. [Log into user 54's account and open the payment history page](https://staging.va.gov/va-payment-history/payments/?postLogin=true).
2. Have developer tools open.
3. Inspect "Check your payment history for your VA disability compensation, pension, and education benefits."
4. Verify it is a h2.
5. Inspect "Payment Benefits."
6. Verify it is a h3.

## Solution

Update the headings into a logical order as demonstrated below. I have included visual heading classes to maintain the intended visual hierarchy.

### Before
```html
<h1>Your VA payments</h1>
  <h2 class="vads-u-font-size--lg vads-u-font-weight--normal vads-u-line-height--5">
    Check your payment history for your VA disability compensation, pension, and education benefits.
  </h2>
    <h3 id="paymentsRecievedHeader" class="vads-u-font-size--xl">
      Payments you received
    </h3>
    <h3 id="paymentsRecievedHeader" class="vads-u-font-size--xl">
      Payments returned
    </h3>
    <h3>
      What if I find a check that I reported missing?
    </h3>
    <h3 class="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary">
      What if I need to change my direct deposit or contact information?
    </h3>
    <!-- Questions continue as h3s -->
```

### After
``` diff
<h1>Your VA payments</h1>
+  <!-- Convert the h2 on this description to a paragraph -->
  <p class="vads-u-font-size--lg vads-u-font-weight--normal vads-u-line-height--5">
    Check your payment history for your VA disability compensation, pension, and education benefits.
  </p>

+ <!-- Convert the h3 payments received and returned to h2s -->
  <h2 id="paymentsRecievedHeader" class="vads-u-font-size--xl">
    Payments you received
  </h2>
  <h2 id="paymentsRecievedHeader" class="vads-u-font-size--xl">
    Payments returned
  </h2>

+ <!-- In the future state when jump links are provided, add frequently asked questions as a heading -->
+ <h2> Frequently Asked Questions</h2>
    <h3>
      What if I find a check that I reported missing?
    </h3>
    <h3 class="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary">
      What if I need to change my direct deposit or contact information?
    </h3>
    <!-- Questions continue as h3s -->
```

## WCAG or Vendor Guidance (optional)

[Headings - Web Accessibility Tutorials](https://www.w3.org/WAI/tutorials/page-structure/headings/)

## Screenshot
This paragraph is currently written as a `h2`
<img width="806" alt="Screen Shot 2020-12-07 at 3 44 51 PM" src="https://user-images.githubusercontent.com/14154792/101405104-970e7b00-38a5-11eb-9751-0838a8f2f1ff.png">

**Comment by jason-gcio** [comment]
Easy enough

**Comment by micahchiang** [comment]
@joshkimux - I've tagged you as a reviewer on the pull request for this ticket - https://github.com/department-of-veterans-affairs/vets-website/pull/15534
